components/rhel/misc/rhel-k8s-singlenode-setup/README.adoc

```diff
@@ -24,15 +24,25 @@ toc::[]
 ----
 $ mkdir directory && cd directory
 ----
-.  Get the latest CDK box and add it to Vagrant. You can download the CDK
-and find further information about CDK
-http://developers.redhat.com/products/cdk/overview/[here]. For detailed instructions on how to install CDK refer to the https://access.redhat.com/documentation/en/red-hat-container-development-kit/2.2/paged/installation-guide[CDK Installation Guide].
 
+.  Get the latest CDK box (using Developer Subscription). You can download
+the CDK and find further information about CDK
+http://developers.redhat.com/products/cdk/overview/[here]. For detailed instructions on how to install CDK refer to the
+https://access.redhat.com/documentation/en/red-hat-container-development-kit/2.2/paged/installation-guide[CDK Installation Guide].
+
+. Add the box to Vagrant as:
 +
-NOTE: The name you assign to the box using the --name parameter in the `$ vagrant box add --name cdkv2 \
-  ~/Downloads/rhel-cdk-kubernetes-7.2-*.x86_64.vagrant-libvirt.box` command, must correspond to the name used by the Vagrantfile used to initialize the box. By default, this is cdkv2 for the Vagrantfiles provided with Container Development Kit.
+----
+$ vagrant box add --name <box-name> <Downloaded Folder>/rhel-cdk-kubernetes-7.2-*.x86_64.vagrant-libvirt.box
+----
+
+. Add the `box-name` as an environment variable:
++
```

Comment:
Should we add a line here saying, 'you can customize the box-name as per your requirements'